nf/main.nf

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env nextflow
+
+include { SENTIEON_DNASCOPE } from './modules/local/sentieon/dnascope/main'
+include { SENTIEON_DNASCOPE as SENTIEON_DNASCOPE_DIPLOID } from './modules/local/sentieon/dnascope/main'
+include { GATK_INTERVALLISTOOLS } from './modules/local/gatk/intervallisttools/main'
+include { PICARD_MERGEVCFS } from './modules/local/picard/mergevcfs/main'
+include { SENTIEON_GVCFTYPER } from './modules/local/sentieon/gvcftyper/main.nf'
+
+workflow {
+    main:
+    alignment = params.alignment ? Channel.fromPath(params.alignment) : Channel.value([])
+    align_index = params.align_index ? Channel.fromPath(params.align_index) : Channel.value([])
```

Comment:
What happens when these aren't provided?

Reply:
Well....they are required. Should I remove the empty channel alternative and/or kill the workflow if missing?

Reply:
Yeah the ternary you are using should only be used for optional inputs.